pytorch_lightning/trainer/logging.py

```diff
@@ -66,13 +66,12 @@ def process_dict_result(self, output, train=False):
             for k, v in output.items():
                 if k in ['log', 'progress_bar']:
                     m = inspect.cleandoc(
-                        f"""The {{{k}:dict keyword}} was deprecated in 0.9.1 and will be removed in 1.0.0
-                        Please use self.log(...) inside the lightningModule instead.
-
-                        # log on a step or aggregate epoch metric to the logger and/or progress bar
-                        # (inside LightningModule)
-                        self.log('train_loss', loss, on_step=True, on_epoch=True, prog_bar=True)
-                    """)
+                        f"The {{{k}:dict keyword}} was deprecated in 0.9.1 and will be removed in 1.0.0"
+                        " Please use self.log(...) inside the lightningModule instead."
+                        " # log on a step or aggregate epoch metric to the logger and/or progress bar"
+                        " (inside LightningModule)\n"
```

Comment:
is this correct?

Reply:
What do you mean?

Reply:
I thought
```
# log on a step or aggregate epoch metric to the logger and/or progress bar
```
this is a comment here and now added to the string. BTW as per TODO this should have been removed in 1.0?

Reply:
well the comment was there even before as before it was text block so including it too